com.viber.Viber.json

```diff
@@ -0,0 +1,73 @@
+{
+  "app-id": "com.viber.Viber",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "viber",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--filesystem=~/.Viber-PC",
```

Comment:
I made this comment before but what did you think about maybe setting `$HOME` to `$XDG_CONFIG_HOME` or such so we keep configuration contained.

Reply:
That seems like a good idea.

Reply:
How were you envisaging doing that? Either I'm doing it wrong (not unexpected) or that doesn't work as well as I'd hoped.

Reply:
Eh don't worry about it I guess.